packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const { getService } = require('../../utils');
+
+/**
+ * Map every stage in the array to be ordered in the relation
+ * @param {Object[]} stages
+ * @param {number} stages.id
+ * @return {Object[]}
+ */
+function buildStagesConnectArray(stages) {
+  return stages.map((stage, index) => {
+    const connect = {
+      id: stage.id,
+      position: {},
+    };
+
+    if (index === 0) {
+      connect.position.start = true;
+    } else {
+      connect.position.after = stages[index - 1].id;
+    }
+    return connect;
+  });
+}
+
+module.exports = ({ strapi }) => {
+  const workflowsService = getService('workflows', { strapi });
+  const stagesService = getService('stages', { strapi });
+
+  return {
+    async bootstrap() {
+      const wfCount = await workflowsService.count();
+      const stagesCount = await stagesService.count();
+
+      // Check if there is nothing about review-workflow in DB
+      // If any, the feature has already been initialized with a workflow and stages
+      if (wfCount === 0 && stagesCount === 0) {
+        const defaultStages = require('../../constants/default-stages.json');
+        const defaultWorkflow = require('../../constants/default-workflow.json');
```

Comment:
In the future, will we allow users to define the default stages / workflows ? 

Just a thought for the future, we could let users define this constants in the plugin configuration. wdyt?

Reply:
This is not a plugin, but surely, we can think of a way to configure the default workflow of RW feature. 👍